CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-rc.5 July 28, 2021
+
+**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+
+## Breaking changes ⚠️
+* Improve camera API consumption from java programming language by adding a JvmName annotations. ([#495](https://github.com/mapbox/mapbox-maps-android/pull/495))
+* Rename AttributionView#setOnClickListener to setViewOnClickListener to avoid overloading the Android SDK method. Results in compilation on Android P and above. Adjust codebase to changes in enforced nullability of Android SDK code. ([#497](https://github.com/mapbox/mapbox-maps-android/pull/497))
+
+## Features ✨ and improvements 🏁
+* Add showLogo and showAttributes config for snapshotter. User can then hide logo and attributions in a snapshotter ([#496](https://github.com/mapbox/mapbox-maps-android/pull/496))
+* Add lifecycle plugin, users don't need to call onStart/onStop/onDestroy methods with this plugin ([#485](https://github.com/mapbox/mapbox-maps-android/pull/485))
+* Add a minimum Android Auto test app and an optional Android Auto extension that provide convenient extension function to initialise the MapSurface from a Car App Session. ([#488](https://github.com/mapbox/mapbox-maps-android/pull/488))
+* Add lint check for lifecycle methods ([#516](https://github.com/mapbox/mapbox-maps-android/pull/516))
+* Get rid of reflection for creating plugins. That will lead to decreasing MapView startup time if plugins are enabled. ([#519](https://github.com/mapbox/mapbox-maps-android/pull/519))
```

Comment:
that's a breaking change, more info in PR description

Reply:
seems it is also marked as feature label, guess that's why it ended up in this section

Reply:
Also, maybe we can add more specifics here about the exact API breaking change.

Reply:
I've marked it with both labels, my fault, script seems to respect only the first label.

Reply:
no worries, we can fix it afterwards anyway 😄